vendor/assets/stylesheets/ms/_variables.scss

```diff
@@ -15,7 +15,7 @@ $letter-spacing-2x: 0.1em;
 
 $default: #333;
 $default-hover: #111;
-$primary: #9E9E9E;
+$primary: #a60505;
 $primary-hover: #ad8b60;
```

Comment:
So, if you change your primary font to a red, you'll want a secondary red-ish color to work as hover. Otherwise, when you hover on something primary you'll get the default burnt gold (unless that's what you want).

Update the `$primary-hover` variable to something else?